staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -425,6 +425,10 @@ type Devices struct {
 	// Defaults to true.
 	// +optional
 	AutoattachMemBalloon *bool `json:"autoattachMemBalloon,omitempty"`
+	// Whether to attach disks for volumes referenced by the VMI.
+	// Defaults to true.
+	// +optional
+	AutoattachVolumeDisk *bool `json:"autoattachVolumeDisk,omitempty"`
```

Comment:
I'm unsure if we need this ability in the current case.

The reason we had the tuning to toggle auto attaching the pod network is because there are use cases where we don't want a network interface at all. Since that feature auto creates both the network and network binding if none are provided, then the boolean was the only way to avoid that.

We also have toggles for things like VNC, Graphics, and serial, but again I believe those toggles exist simply because there's no way to disable the feature otherwise.

For disks, we'd only need this ability disable automatic volume matching if it is possible to define a volume that should not have a corresponding disk/filesystm associated with it. Is there such a use case?